<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>952</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>20</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Intervals:</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputGraph">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>381</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>212</width>
      <height>545</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>X = [</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="inputLeftX">
                 <property name="text">
                  <string>-10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>,</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inputRightX">
                   <property name="text">
                    <string>10</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>]</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Y = [</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="inputLeftY">
                 <property name="text">
                  <string>-10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>,</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inputRightY">
                   <property name="text">
                    <string>10</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>]</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Accuracy:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inputAccuracy">
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>0.1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="PROGA34">
         <property name="text">
          <string>34 прога</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QCheckBox" name="Bessel">
         <property name="text">
          <string>Бессель</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QCheckBox" name="main">
             <property name="maximumSize">
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <italic>false</italic>
               <underline>false</underline>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Ньютон&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QCheckBox" name="sin">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>&lt;font color=green&gt;sin</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QCheckBox" name="cos">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>&lt;font color=red&gt;cos</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QCheckBox" name="tg">
             <property name="maximumSize">
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>&lt;font color=darkMagenta&gt;sin(1/x)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Ввод из файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Ввод по формуле 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Ввод по формуле 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Порядок производной:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DerDeg">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Масштаб</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputScale">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Количество точек:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputNumber">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>где добавить дельту</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="InputDelta"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Повернуть на 90 градусов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="draw">
         <property name="text">
          <string>Draw</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Сохранить картинку</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>inputRightX</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputRightX</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputRightX</sender>
   <signal>selectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>895</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputRightX</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>873</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputLeftX</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>867</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputLeftX</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputAccuracy</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>855</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputLeftY</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputRightY</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>823</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DerDeg</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>855</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputScale</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>851</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputNumber</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>856</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>RadioClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>929</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>RadioClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>RadioClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>844</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTextChanged()</slot>
  <slot>RadioClick()</slot>
 </slots>
</ui>
